cmd/export.go

```diff
@@ -44,7 +44,7 @@ var (
 		Long:              "Export output values from deployment group to other deployment groups that depend upon them.",
 		Args:              cobra.MatchAll(cobra.ExactArgs(1), checkDir),
 		ValidArgsFunction: matchDirs,
-		PreRun:            setArtifactsDir,
+		PreRunE:           parseExportImportArgs,
```

Comment:
Why `PreRunE`? `parseExportImportArgs` never errors.

Reply:
I converted the rest of the functions to use PreRunE (which I *think* has stricter connection between exit code and GoLang `error`) but didn't notice this one is kinda silly. I've converted back to `PreRun`